mlflow/tracing/provider.py

```diff
@@ -109,7 +109,30 @@ def _force_set_otel_tracer_provider(tracer_provider):
 
 def disable():
     """
-    Disable tracing by setting the global tracer provider to NoOpTracerProvider.
+    Disable tracing.
```

Comment:
```suggestion
    Disable tracing. <extra info>
```

Any extra info or phrase we could add here?

Reply:
Can we add back the original saying `by setting the global tracer provider to NoOpTracerProvider.` It's useful for devs I think, maybe not for customers

Reply:
@serena-ruan Good question. My concern it makes users wonder what `NoOpTracerProvider` is?

Reply:
@B-Step62 WDYT?

Reply:
Hmm I think both perspectives are fair points.... What about adding link to https://github.com/open-telemetry/opentelemetry-python/blob/4febd337b019ea013ccaab74893bd9883eb59000/opentelemetry-api/src/opentelemetry/trace/__init__.py#L222

```
Disable tracing.

This function configures OpenTelemetry to use `NoOpTracerProvider <https://github.com/open-telemetry/opentelemetry-python/blob/4febd337b019ea013ccaab74893bd9883eb59000/opentelemetry-api/src/opentelemetry/trace/__init__.py#L222>`_  for preventing any traces  from being generated.
```
(^ the wording could be improved)